charts/alumet/charts/alumet-relay-client/templates/daemonset-alumet-relay-client.yaml

```
@@ -0,0 +1,163 @@
+apiVersion: apps/v1
+kind: DaemonSet
+metadata:
+  name: {{ template "alumet-relay-client.fullname" . }}
+
+  
+spec:
+  selector:
+    matchLabels:
+      name: alumet-relay-client
+  template:
+    metadata:
+      labels:
+        name: alumet-relay-client
+    spec:
+      tolerations:
+      # these tolerations are to have the daemonset runnable on control plane nodes
+      # remove them if your control plane nodes should not run pods
+      #- key: kubernetes.io/hostname
+      # operator: Equal
+      # value: "hostname"
+      # effect: NoSchedule
+      - key: node-role.kubernetes.io/master
+        operator: Exists
+        effect: NoSchedule
+      - key: master
+        operator: Exists
+        effect: NoSchedule
+
+      serviceAccountName: {{ .Release.Name }}-alumet-reader
+      automountServiceAccountToken: true
+
+      initContainers:
+        - name: wait-for-alumet-server
+          image: docker.io/busybox:latest
+          command: [
+            'sh',
+            '-c',
+            'until nc -vz {{ .Release.Name }}-alumet-relay-server {{ .Values.plugins.relay_client.port }}; do echo "Waiting for alumet-relay-server to be running"; sleep 2; done;'
+          ]
+
+      containers:
+      - name: alumet-relay-client
```

Your yaml syntax isn't consistent

```suggestion
        - name: alumet-relay-client
```

---

done